lib/manager/gradle-lite/readme.md

```diff
@@ -0,0 +1 @@
+This is an alternate manager for Gradle, written completely in JavaScript.
```

Comment:
We should add some more description, so users are welcome to disable default `gradle` and test this new manager, as we plan to replace to old in future.

Reply:
My additions are not ideal, but at least someone can correct me instead of writing from scratch.

Reply:
Maybe @HonkingGoose  has some time to review this 🙃